src/bin/poudriere-jail.8

```diff
@@ -89,155 +85,222 @@ Except for
+.Pq as if Fl -depth=1 No was specified to the Git command .
+.It Fl f Ar filesystem
+Specify the
+.Ar filesystem
+name
+.Pq ${ Ns Va ZPOOL Ns } Ns Pa /jails/filesystem .
+.It Fl J Ar number
+Run
 .Ar number
-of make jobs will run in parallel for buildworld.
-Defaults to the number of CPUs reported by:  sysctl hw.ncpu.
-.It Fl q
-Remove the header when
-.Fl l
-is the specified mandatory option.
-Otherwise, it has no effect.
+of
+.Xr make 1
+jobs in parallel for buildworld.
+Defaults to the number of CPUs reported by:
+.Dq Li sysctl hw.ncpu .
 .It Fl j Ar name
-Specifies the
+Specify the
 .Ar name
 of the jail.
-.It Fl v Ar version
-Specifies which
-.Ar version
-of
-.Fx
-to use in the jail.
-If you are using method ftp then the
-.Ar version
-should in the form of: 9.0-RELEASE.
-If you are using method of svn then the
-.Ar version
-should be in the form of git or svn branches: stable/9 or head for CURRENT.
-Other methods only use the value for display.
-.It Fl a Ar architecture
-Specifies which
-.Ar architecture
-of
-.Fx
-to use in the jail.
-(Default: same as host)
+.It Fl K Ar kernelname
+Install the jail with the specified kernel name.
+If the
+.Ar kernelname
+is an empty string then GENERIC will be used.
+If installing from FTP then the default kernel is installed and
+the value of
+.Ar kernelname
+is ignored.
+.It Fl M Ar mountpoint
+Use an alternative
+.Ar mountpoint
+when creating the jail.
 .It Fl m Ar method
-Specifies which
+Specify which
 .Ar method
 to use to create the jail.
-(default:
-.Sy http )
+The default is
+.Cm http .
 .Pp
 Pre-built distribution options:
-.Bl -tag -width "ftp-archiveXX"
-.It Sy allbsd
-Use
-.Lk http://www.allbsd.org.
-.It Sy ftp Sy http
-Fetch from configured
-.Sy FREEBSD_HOST
-variable from
+.Bl -tag -width "ftp-archive"
+.It Cm allbsd
+Fetch from
+.Lk http://www.allbsd.org .
+.It Cm ftp
+Fetch from the host specified to the
+.Va FREEBSD_HOST
+variable in
 .Pa poudriere.conf .
-.It Sy ftp-archive
+.It Cm ftp-archive
 Fetch from
-.Lk http://ftp-archive.freebsd.org.
-.It Sy null
-This option can be used to import an existing directory that already contains an installed system.
+.Lk http://ftp-archive.freebsd.org .
+.It Cm http
+See
+.Cm ftp .
+.It Cm null
+This option can be used to import an existing directory that already contains
+an installed system.
 The path must be specified with
 .Fl M Ar path .
 It is expected that this directory be installed to with the following:
-.Bd -literal -offset indent
-/usr/src# make installworld DESTDIR=PATH DB_FROM_SRC=1
-/usr/src# make distrib-dirs DESTDIR=PATH DB_FROM_SRC=1
-/usr/src# make distribution DESTDIR=PATH DB_FROM_SRC=1
+.Bd -literal -offset 2n
+.Ic # cd /usr/src
+.Ic # Ic make installworld DESTDIR= Ns Ar PATH Ic DB_FROM_SRC=1
+.Ic # make distrib-dirs DESTDIR= Ns Ar PATH Ic DB_FROM_SRC=1
+.Ic # make distribution DESTDIR= Ns Ar PATH Ic DB_FROM_SRC=1
 .Ed
 .Pp
 The path will be null-mounted during builds.
```

Comment:
Thanks for your PR. You have some stuff to cleanup. In the meantime I found that this line is not accurate and removed it. Sorry for the conflict.

Reply:
Git handled this correctly! There's nothing for me to clean up!